cai_causal_graph/time_series_causal_graph.py

```diff
@@ -432,6 +440,15 @@ def extend_graph(
             maxlag = minimal_graph.max_backward_lag
             assert maxlag is not None
 
+            # create all the nodes from 0 to backward_steps
```

Comment:
Why is this change needed? Was this a bug too? It doesn't seem to be picked up in the changelog, so I would recommend to comment on it

Reply:
No, I don't need it. It was a testing leftover. Reverted.